### PR TITLE
fix(config): updating cache-server url to stable

### PR DESCRIPTION
**Pull request by jrhender**
This is because for asset claims a connection over NATS protocol is required
This is only open on the dev cache server which now requires authentication

### CODE REVIEW

File: src/config/config.volta.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ export const config: IOcnToolsConfig = {
             cacheServerUrl: "https://volta-iam-cacheserver.energyweb.org/",
```
Comment (adamstaveley) [inline src/config/config.volta.ts]: does this one need updating too?

Comment (adamstaveley) [reply src/config/config.volta.ts]: although I guess this isn't used really

**Review by adamstaveley — DISMISSED** [review]

File: src/config/config.volta.ts (hunk 1 above)
Comment (jrhender) [reply src/config/config.volta.ts]: Fixed with new commit

**Review by adamstaveley — APPROVED** [review]
LGTM

**Pull request merged**